Build/Common.Build.Default.props

```diff
@@ -43,7 +43,6 @@
   <!-- Change configuration name to output to a different folder -->
   <!-- POGO instrumentation -->
   <PropertyGroup>
-    <IsPogoBuild />
```

Comment:
That occurred to me and I wasn't sure but it hasn't seemed to cause any problems. I think at the beginning of any builds all properties are undefined which is the same as default-initializing to blank. I already checked it into release/1.2 and it hasn't caused any problems AFAICT. I need to do a deeper investigation before I commit to being sure about that, though. I'll revert it in this change and in the other branch as well.